backend/app/auth/jwt.py

```
@@ -0,0 +1,24 @@
+from datetime import datetime, timedelta
+from jose import jwt, JWTError
+from app.core.config import settings
+
+def create_access_token(data: dict, expires_delta: int = None):
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)